src/services/AuthAPI.ts

```diff
@@ -32,4 +32,27 @@ async function loginUser(
   }
 }
 
-export { loginUser };
+async function logoutUser(): Promise<
+  | {
+      ok: true;
+    }
+  | {
+      ok: false;
+      message: string;
+    }
+> {
+  let resp = await fetch(`${BASE_URL}/api/auth/logout`, {
+    method: "POST",
+  });
+  if (resp.ok) {
+    return {
```

Comment:
You should get the JSON resp, and put the message on the returned object. Also make sure to change the TS type above.